src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -641,6 +644,7 @@ public void windowClosing(WindowEvent e) {
             bp.updateSearchManager();
             // Set correct enabled state for Back and Forward actions:
             bp.setBackAndForwardEnabledState();
+            bp.getUndoManager().postUndoRedoEvent();
```

Comment:
The undoManager has the event bus. Any other suggestions for the location
of the undo event bus?